src/components/table/index.tsx

```diff
@@ -201,7 +206,7 @@ export default function EnhancedTable<T>({
   return (
     <>
       {showTopBar && <EnhancedTableToolbar numSelected={selectedRows ? selectedRows.length : 0} topBarButtons={topBarButtons} />}
-      <TableContainer id={id}>
+      <TableContainer id={id} ref={containerRef}>
         <DndContext sensors={sensors} onDragEnd={handleDragEnd}>
           <Table stickyHeader={true} aria-labelledby='tableTitle' size='medium' aria-label='enhanced table' className={classes.table}>
```

Comment:
I tried forever to get the built-in `stickyHeader` functionality to work. The best I could do involved fixing the height of the table container, which produced a gross double-scrollbar effect.

Reply:
However, if anyone has any guidance on how to make that work it could be a lot simpler than what I implemented here, and also hopefully do an actual sticky effect instead of floating in like what mine does.

Reply:
This seems to work fine in the vercel preview.

There is another way to make this work if you want to try that and then decide which one might be better...

1) change thead to TableHead (like you have already done) here https://github.com/terraware/web-components/blob/main/src/components/table/TableHeader.tsx#L38 - I don't have enough context on why we are using thead here instead of TableHead, we could check with @constanzauanini to see if she remembers
2) no changes here https://github.com/terraware/web-components/blob/main/src/components/table/index.tsx#L206 (see `stickyHeader={true}` option that supports a sticky header)
3) setting a fixed height on this https://github.com/terraware/web-components/blob/main/src/components/table/index.tsx#L204 (now that's the tricky part.. ) - will enable sticky header to work properly

Reply:
I remember thead was related to allow dragging columns to reorder them. If that still works with TableHead, then we can change it. We can test that on accession table, or maybe we should make the table story drag to work, so we can test it there

Reply:
With a change to TableHead, dragging and dropping the column headers seems to work, as well as clicking to change the sort.